### DISCUSSION TITLE
AnimeDex v1.5.1

### DISCUSSION THREAD

**Discussion by TechShreyash** [TechShreyash/AnimeDex#22] [Announcements]
## ⭐ Powered By TechZApi

- Now you need add your TechZApi Key in config.py or in env
- Get your api key from @TechZApiBot

## ♻️ Updates

- Download Links in multiple qualities on episode page
- Video Player fixed
- And many more other fixes and improvements

**Full Changelog**: https://github.com/TechShreyash/AnimeDex/compare/v1.4.2...v1.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/TechShreyash/AnimeDex/releases/tag/v1.5.1'>AnimeDex v1.5.1</a>.</em>

**Comment by Buttersxxx** [comment]
**I'm really liking this, it's working Very well **

**Comment by TechShreyash** [comment]
Enjoy 👍
